Back out "[pytorch] use cublas lt interface for bias fusion"

jspark1105 opened the pull request:
Summary:
Original commit changeset: cecc51a27f4b

Original Phabricator Diff: D33928147 (https://github.com/pytorch/pytorch/commit/3aecce70152579a01e969b8ea1540f301290fe5a)

We are seeing a couple of issues with cublas lt. Sending this diff in case we need to revert.

Differential Revision: D34486240

jasonjk-park commented:
Found a case where cublasLt fails with CUBLAS_STATUS_EXECUTION_FAILED.

(1000x10) * (10x10) + bias(10) works.
(655360x10) * (10x10) + bias(10) fails.
disabling cublasLt, (655360x10) * (10x10) + bias(10) works.

jspark1105 committed:
Back out "[pytorch] use cublas lt interface for bias fusion" (#73455)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/73455

Original commit changeset: cecc51a27f4b

Original Phabricator Diff: D33928147 (https://github.com/pytorch/pytorch/commit/3aecce70152579a01e969b8ea1540f301290fe5a)

We are seeing a couple of issues with cublas lt. Sending this diff in case we need to revert.

For example, we're observing
(1000x10) * (10x10) + bias(10) works.
(655360x10) * (10x10) + bias(10) fails.
disabling cublasLt, (655360x10) * (10x10) + bias(10) works

Differential Revision: D34486240

fbshipit-source-id: 3cd44d5d5d650c483beee751bd8047a13d3683e3

jasonjk-park commented:
error was on A100
